source/contact.erb

```diff
@@ -0,0 +1,104 @@
+---
+title: "get in touch..."
+subtitle: "to arrange a meeting or have a chat, just drop us a line"
+
+offices:
+
+  - name: "London"
+    address: |
+      17 Blossom Street
+      E1 6PL
+      London, UK
+    humanized_phone_number: "+44 20 7183 4250"
+    phone_number: "+442071834250"
+    email: "hello@unboxed.co"
+    map_link: "http://goo.gl/maps/AuKrs"
+    map_image: "https://a.tiles.mapbox.com/v4/web-unboxed.oebohej0/attribution,zoompan,zoomwheel,geocoder,share.html?access_token=pk.eyJ1Ijoid2ViLXVuYm94ZWQiLCJhIjoiY2lpN2hneWlvMDA4YnYzbHg2NHFqbW9yaSJ9.J1ksUtXP8Xsb2piQUcWJfA"
+
+    transport_directions: "nearest stations"
+    station_and_parking:
+      - name: "liverpool street"
+        link: "https://tfl.gov.uk/tube/stop/940GZZLULVT/liverpool-street-underground-station"
+        icon: "tfl-icon__underground.png"
+      - name: "shoreditch highstreet"
```

Comment:
As we should use these names for the `alt` attribute too, we should probably capitalise them as they are place - we can then lowercase by CSS using the `text-transform` property